all-actors/src/main/java/org/sunbird/serviceimpl/CertsServiceImpl.java

```diff
@@ -221,8 +224,16 @@ public Response downloadV2(Request request) throws BaseException {
         if (CollectionUtils.isNotEmpty(resultList) && MapUtils.isNotEmpty(resultList.get(0))) {
             Map<String, Object> certInfo = resultList.get(0);
             try {
-                Map<String, Object> data = requestMapper.readValue((String) certInfo.get(JsonKeys.DATA), new TypeReference<Map<String, Object>>() {});
-                response.put(JsonKeys.PRINT_URI, data.get(JsonKeys.PRINT_URI));
+                String jsonUrl = (String) certInfo.get(JsonKeys.JSON_URL);
+                String printUri;
+                if (StringUtils.isEmpty(jsonUrl)) {
```

Comment:
pls leave a comment why this will not be empty...we will not know these details otherwise.

Reply:
done